connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoSinkOptions.scala

```diff
@@ -76,7 +76,7 @@ object KustoSinkOptions extends KustoOptions {
   val KUSTO_TEMP_TABLE_NAME: String = newOption("tempTableName")
 
   // The batch size that we want to use for each "streaming batch". The default is 4MB.
```

Comment:
You used the word batch twice for two things. Batch is a known name in Spark structured streaming for a single call to a sink via "addBatch()"
```suggestion
  // The chunk size that we want to use while iterating over a "streaming batch". The default is 4MB.
```